SYCL/ESIMD/lsc/atomic_smoke.cpp

```diff
@@ -658,12 +658,14 @@ int main(void) {
 
   // Check load/store operations
   passed &= test_int_types<8, ImplLoad>(q, cfg);
+#ifndef USE_SCALAR_OFFSET
   if (q.get_backend() != sycl::backend::ext_intel_esimd_emulator)
     passed &= test_int_types<8, ImplStore>(q, cfg);
 #ifndef USE_DWORD_ATOMICS
   if (q.get_backend() != sycl::backend::ext_intel_esimd_emulator)
     passed &= test<float, 8, ImplStore>(q, cfg);
 #endif // USE_DWORD_ATOMICS
+#endif
```

Comment:
nit (here and in few other places):
```suggestion
#endif // !USE_SCALAR_OFFSET
```